draft-ietf-scitt-software-use-cases.md

```diff
@@ -277,7 +277,7 @@ A consumer of a released software wants:
 
 There is no standardized way to:
 
-- discover and associate relevant documents and check results required for various types of audits
+- discover and associate relevant documents, data, and check results required for various types of audits
```

Comment:
Minor, non-blocking: most auditors have documents that describe conformance to a requirement, and then they review not only supporting documents, but supporting data (I can joke it is too often more the former than the latter, but I digress). We are interested in may opaque payloads, many are data, not really "just" documents. Not sure if this will help others, but it would help me and maybe others without getting too technical.